src/ralph/export_to_ng/publishers.py

```diff
@@ -81,8 +91,29 @@ def sync_device_to_ralph3(sender, instance=None, created=False, **kwargs):
         'service': device.service.uid if device.service else None,
         'environment': device.device_environment_id,
         'venture_role': device.venture_role_id,
+        'custom_fields': {
+            k: v for k, v in device.get_property_set().items()
+            if k in settings.RALPH2_HERMES_ROLE_PROPERTY_WHITELIST
+        },
     }
-    return data
+    return {k: v for k, v in data.items()} if fields else data
```

Comment:
are you sure this is working? :) what's the difference between `{k: v for k, v in data.items()}` and `data`? :)